torch/testing/_internal/common_methods_invocations.py

```diff
@@ -797,6 +797,8 @@ class OpInfo(object):
 
     supports_expanded_weight: bool = False
 
+    is_factory_function: bool = False
```

Comment:
Maybe there is a better way to infer that its a factory function?

Reply:
this seems about as good as it gets